src/IB/IBFEMethod.cpp

```diff
@@ -319,6 +322,58 @@ libmesh_restart_file_name(const std::string& restart_dump_dirname,
                      << std::setfill('0') << std::right << time_step_number << "." << extension;
     return file_name_prefix.str();
 }
+
+template <int DIM>
+class CopyAsRefineOperator : public xfer::RefineOperator<DIM>
```

Comment:
What is this meant to do?

Reply:
Illustrate my lack of SAMRAI knowledge :)